src/map/atcommand.cpp

```diff
@@ -8262,11 +8268,16 @@ ACMD_FUNC(iteminfo)
 		sprintf(atcmd_output, msg_txt(sd,269), MAX_SEARCH); // Displaying first %d matches
 		clif_displaymessage(fd, atcmd_output);
 	}
+
+	struct item tmp_item = {};
+
 	for (const auto &result : item_array) {
 		std::shared_ptr<item_data> item_data = result.second;
-
+		itemid = item_data->nameid;
+		tmp_item.nameid = itemid;
+		
 		sprintf(atcmd_output, msg_txt(sd,1277), // Item: '%s'/'%s'[%d] (%u) Type: %s | Extra Effect: %s
-			item_data->name.c_str(),item_data->ename.c_str(),item_data->slots,item_data->nameid,
+			createItemLink(tmp_item).c_str(),item_data->nameid,
```

Comment:
When I tested a few years ago, the item name was unreadable if the item didn't exist in the client. Do you know if there is a way to check if the item exists in the client serverside?

Reply:
> When I tested a few years ago, the item name was unreadable if the item didn't exist in the client. Do you know if there is a way to check if the item exists in the client serverside?

Sorry, no